### ISSUE TITLE
difference between your experiment and my reproduction

### ISSUE THREAD

**Issue by Ryngar** [William-wAng618/M2PT#5]
Following your README，I conduct an experiment but got about 900 MME score which is 1490 in your paper after M2PT .
I wonder if there is any mistake?

**Comment by William-wAng618** [comment]
Thanks for your interesting to our work, based on the limited information and my knowledge, this may come from:
1. Please check prompt weights, if they are not load properly.  Will yield very low results
2. Please check the evaluation scripts, make sure the same version as our latest uploaded
3. Please check the training loss curve, see if the prompt is correctly updated.

And you can find the well-trained prompts (10-20) in this link: https://drive.google.com/file/d/1cG0y1Nc5eYI1Atn-f0PLsDBi77YLlVYs/view?usp=sharing

Good luck!

**Comment by qmechlab** [comment]
Hi,

Thank you for sharing the link to the pre-trained weights.

However, I encountered an error when trying to unzip the file. The archive appears to be corrupted — specifically, `pytorch_model-00001-of-00002.bin` fails the CRC check during extraction (both on Linux and macOS). 

Could you please re-upload the zip file to Google Drive? It's possible the file was corrupted during the upload process.

Thank you!

<img width="378" height="138" alt="Image" src="https://github.com/user-attachments/assets/83a7454b-a5a8-4417-9831-7b4f19ae7330" />
